src/lib/ui/ProBeneficiaryCreate/beneficiary.schema.ts

```diff
@@ -1,26 +1,23 @@
-import { isDate } from 'date-fns';
+import { validateCodePostal, validateDateInput, validatePhoneNumber } from '$lib/validation';
 import * as yup from 'yup';
 
 export const beneficiaryAccountSchema = yup.object().shape({
 	firstname: yup.string().required(),
 	lastname: yup.string().required(),
-	dateOfBirth: yup.string().test('is-date-valid', 'Le format de la date est incorrect', (value) => {
-		if (!/[0-9]{4}-[0-9]{2}-[0-9]{2}/.test(value)) {
-			return false;
-		}
-		const date = new Date(value);
-		return isDate(date);
-	}),
+	dateOfBirth: yup
+		.string()
+		.test('is-date-valid', 'Le format de la date est incorrect', (value) => {
+			return validateDateInput(value);
+		})
```

Comment:
suggestion: Est-ce qu'il y a une raison de pas faire plutôt ça ?

```suggestion
		.test('is-date-valid', 'Le format de la date est incorrect', validateDateInput)
```